front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/Settings/index.tsx

```diff
@@ -20,8 +20,11 @@ const customPageSlugAllowedToEdit: { [key in TCustomPageCode]: boolean } = {
   about: true,
   custom: true,
   faq: false,
+  // These pages shouldn't appear on the Pages & menu page at the moment.
```

Comment:
When I checked TCustomPageCode usage, I got an error about an entry for cookie-policy that was missing. Not sure why it didn't show up via CI. 🤔